tests/tensor/rewriting/test_linalg.py

```diff
@@ -545,3 +545,25 @@ def test_svd_uv_merge():
             assert node.op.compute_uv
             svd_counter += 1
     assert svd_counter == 1
+
+
+@pytest.mark.parametrize("inv_op_1", ["inv", "pinv", "solve", "solve_triangular"])
+@pytest.mark.parametrize("inv_op_2", ["inv", "pinv", "solve", "solve_triangular"])
+def test_inv_inv_rewrite(inv_op_1, inv_op_2):
+    def get_pt_function(x, op_name):
+        if "solve" in op_name:
+            return getattr(pt.linalg, op_name)(x, pt.eye(x.shape[0]))
+        return getattr(pt.linalg, op_name)(x)
+
+    x = pt.matrix("x")
+    op1 = get_pt_function(x, inv_op_1)
+    op2 = get_pt_function(op1, inv_op_2)
+    f_rewritten = function([x], op2, mode="FAST_RUN")
+    print(f_rewritten.dprint())
+    nodes = f_rewritten.maker.fgraph.apply_nodes
```

Comment:
This is so simple that I would just call `rewrite_graph` and assert the output is x again.